modules/monitoring-components-for-monitoring-user-defined-projects.adoc

```diff
@@ -0,0 +1,29 @@
+:_module-type: CONCEPT
+// Module included in the following assemblies:
+//
+// * assemblies/understanding-the-monitoring-stack.adoc
+
+[id="components-for-monitoring-user-defined-projects_{context}"]
+= Components for monitoring user-defined projects
+
+[role="_abstract"]
+{product-title} includes an optional enhancement to the monitoring stack that enables you to monitor services and pods in user-defined projects. This feature includes the following components:
+
+.Components for monitoring user-defined projects
+[options="header"]
+|===
+
+|Component|Description
+
+|Prometheus Operator
+|The Prometheus Operator (PO) in the `openshift-user-workload-monitoring` project creates, configures, and manages Prometheus and Thanos Ruler instances in the same project.
+
+|Prometheus
+|Prometheus is the monitoring system through which monitoring is provided for user-defined projects. Prometheus sends alerts to Alertmanager for processing.
```

Comment:
Maybe a note here about the lack of alert routing

Reply:
Done.